pkg/distro/rhel/rhel9/azure.go

```diff
@@ -490,6 +490,11 @@ func defaultAzureImageConfig(rd *rhel.Distribution) *distro.ImageConfig {
 					},
 				},
 			)
+
+			ic.DisabledServices = append(
+				ic.DisabledServices,
+				"microcode.service",
+			)
```

Comment:
Welp.  We're gonna need an architecture switch.  This is x86 only.

Reply:
Actually, I think we could (or should?) be excluding `microcode_ctl` entirely from Azure images.